IdentityCore/src/webview/systemWebview/MSIDSafariViewController.h

```diff
@@ -26,15 +26,16 @@
 //------------------------------------------------------------------------------
 
 #import <Foundation/Foundation.h>
-#import "MSIDWebviewRequest.h"
+#import "MSIDSystemWebviewController.h"
 
-@class MSIDRequestParameters;
+@interface MSIDSafariViewController : NSObject
 
-@interface MSIDEmbeddedWebviewRequest : MSIDWebviewRequest
+- (id)initWithURL:(NSURL *)url
+          context:(id<MSIDRequestContext>)context;
 
-- (id)initWithRequestParameters:(MSIDRequestParameters *)requestParameter;
+- (BOOL)handleURLResponse:(NSURL *)url;
+- (BOOL)start;
 
-@property NSURL *endURL;
-@property NSDictionary *headers;
+@property id<MSIDSystemWebviewResponseDelegate> webviewDelegate;
```

Comment:
where is webviewDelegate set? Also, if it will be set in a "wrapper" class, who has reference to this controller, this should be weak.